vue-frontend/config/prod.env.js

```diff
@@ -4,4 +4,8 @@ module.exports = {
   // BASE_API: '"http://localhost:5000/"',
   BASE_API: '"https://calibration-faucet-api.filswan.com/"',
   BASE_NETWORK: '"https://matic-mumbai.chainstacklabs.com/"',
+  TOKEN_ADDRESS: '"0xe11A86849d99F524cAC3E7A0Ec1241828e332C62"',
+  FAUCET_ADDRESS: '"0xd5b31FB565d608692d6422beB31Bf0875dad4fC3"',
+  MATIC_TOKEN_ADDRESS: '"0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE"',
+  GOOGLE_RECAPTCHA_KEY: '"6LcKyLYbAAAAANM74ESqR5Q7Z_W2yolLdyK3fzFP"'
```

Comment:
normally keep this key info on your local, putting it onto public repo is a big risk